lib/pageflow/configuration.rb

```diff
@@ -438,7 +439,7 @@ def paperclip_filesystem_root=(_val)
     end
 
     def revision_components
-      page_types.map(&:revision_components).flatten.uniq
+      @revision_components
     end
```

Comment:
Let's add `revision_components` to the `attr_reader`s at the top of the class. Let's also add a doc string since this will now be part of the public API.